app/src/organisms/ChooseRobotToRunProtocolSlideout/index.tsx

```diff
@@ -83,19 +97,10 @@ export function ChooseRobotToRunProtocolSlideoutComponent(
 
   const offsetCandidates = useOffsetCandidatesForAnalysis(
     mostRecentAnalysis,
-    selectedRobot?.ip ?? null
+    null
   )
 
-  const { uploadCsvFile } = useUploadCsvFileMutation(
-    {},
-    selectedRobot != null
-      ? {
-          hostname: selectedRobot.ip,
-          requestor:
-            selectedRobot?.ip === OPENTRONS_USB ? appShellRequestor : undefined,
-        }
-      : null
-  )
+  const { uploadCsvFile } = useUploadCsvFileMutation({}, {})
```

Comment:
Question/nit, can we just pass nothing here?
```suggestion
  const { uploadCsvFile } = useUploadCsvFileMutation()
```